Classes/Hooks/DataHandlerHook.php

```diff
@@ -121,10 +121,11 @@ private function processSingleEvent(int $uid)
         $this->copyDatesFromTimeSlots($uid, $updatedData);
         $this->sanitizeEventDates($updatedData);
 
+        /** @var HookProvider */
```

Comment:
`@var` annotations for local variables always need the variable name (unlike the annotation for fields):

`/** @var HookProvider $dataSanitizationHookProvider*/`